docs/core/tools/troubleshoot-usage-issues.md

```diff
@@ -153,6 +160,51 @@ Tool installation requires access to the NuGet feed that contains the tool packa
 
 A common reason for failure is that the tool name isn't correct. This can happen because of mistyping, or because the tool has moved or been deprecated. For tools on NuGet.org, one way to ensure you have the name correct is to search for the tool at NuGet.org and copy the installation command.
 
+### 401 (Unauthorized)
+
+Most likely you've specified an alternative NuGet feed, and that feed requires authentication. There are a few different ways to solve this:
+
+01. Add the [`--ignore-failed-sources`](dotnet-tool-install.md#arguments) parameter to bypass the error from the private feed and use the public Microsoft feed.
```

Comment:
Since these are alternatives rather than sequential steps, consider making these bullets instead of numbers.